File: geometry/arc_area/kotlin/arc_area.kt
/**
* Calculates the area of arc given by the angle of the arc
*
* @param radius the radius of the circle
* @param angle the angle of the arc in degree
* @return a Float value of the specified
*/

fun getArcArea(radius : Float, angle : Float) =
    (angle/360)*Math.PI*(radius*radius)

fun main(args : Array<String>) {
    val area = getArcArea(4.0f, 37.0f)
    println(area)
}

File: geometry/arc_area/cpp/arc_area.cpp
#define _USE_MATH_DEFINES
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int radius, angle;
    cin >> radius >> angle;
    float area = (float(angle) / 360) * (M_PI * (float(radius) * float(radius)));
    cout << area << "\n";
}
